docs/core/compatibility/3.1-5.0.md

```diff
@@ -254,6 +254,7 @@ If you're migrating from version 3.1 of .NET Core, ASP.NET Core, or EF Core to v
 
 - [WinHttpHandler removed from .NET runtime](#winhttphandler-removed-from-net-runtime)
 - [MulticastOption.Group doesn't accept a null value](#multicastoptiongroup-doesnt-accept-a-null-value)
+- [Cookie Path handling now conforms to RFC 6265](#cookie-path-handling-now-conforms-to-rfc-6265)
```

Comment:
Please also add the include link to the networking category file: https://github.com/dotnet/docs/blob/master/docs/core/compatibility/networking.md